pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -164,6 +164,13 @@ type KeycloakAPIRealm struct {
 	// Roles
 	// +optional
 	Roles *RolesRepresentation `json:"roles,omitempty"`
+
+	// Scope Mappings
+	// +optional
+	ScopeMappings []ScopeMappingRepresentation `json:"scopeMappings,omitempty"`
+	// Client Scope Mappings
```

Comment:
nitpick: new line

Reply:
I consider those two a group, if you want me to change it anyway let me know.